docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/amazon-eks.md

```diff
@@ -31,6 +31,43 @@ that uses the Amazon EBS `gp3` volume type. Then, use it cluster-wide as a defau
 If you encounter issues with EBS CSI Driver, follow the instructions in the [helm-profiles repository](https://github.com/camunda-community-hub/camunda-8-helm-profiles/blob/main/aws/README.md#ebs-csi-driver-addon) maintained by the Camunda Consulting Team.
 :::
 
+## Load Balancer Setup
+
+AWS is offering different types of Load Balancers (LB). Those namely being Classic Load Balancer (CLB), Network Load Balancer (NLB), and Application Load Balancer (ALB). Typically the NLB and ALB are used in production setups.
+
+The zeebe-gateway requires gRPC to work, which in itself requires http2 to be used. Additionally, it's recommended to secure the endpoint with TLS.
```

Comment:
Maybe drop a link to https://grpc.io/ for people wondering why it is so special.